build/distributions/CentOS/6/x86_64/yum_data/packages.yaml

```diff
@@ -223,13 +223,10 @@ qstat:
   :source: https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/6/qstat-2.11-9.20080912svn311.el6.x86_64.rpm
 radiusclient-ng:
   :rpm_name: radiusclient-ng-0.5.6-5.el6.x86_64.rpm
-  :source: http://mirror.cogentco.com/pub/linux/epel/6/x86_64/radiusclient-ng-0.5.6-5.el6.x86_64.rpm
-razor-server:
-  :rpm_name: razor-server-1.6.1-1.el6.noarch.rpm
-  :source: https://yum.puppetlabs.com/el/6Server/PC1/x86_64/razor-server-1.6.1-1.el6.noarch.rpm
+  :source: http://lug.mtu.edu/epel/6/x86_64/Packages/r/radiusclient-ng-0.5.6-5.el6.x86_64.rpm
```

Comment:
python-six:  Why is this in 6_x_Alpha_Dependencies and not 6_X_Dependencies?  Does this mean we don't need it?

Reply:
python-six was only listed in CentOS 6 packages.yaml. Assuming it was removed, just not from this one. Removed it and will re-add if any issues arise.